toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
> Having said that, the goal of a localization comment is to explain what the final result will be to a localizer, and I think saying that $status is a number would only make it confusing, because what localizers care about is that a string will be displayed between parenthesis, not the number. 

How about this example:

```
unreadEmails = You have { PLURAL($num) -> 
  [0] no unread emails
  [one] one unread email
 *[other] many unread emails
}
```

How would you describe the `$num` variable? It's clearly a string but we're going to run `PLURAL` on it and select a string that will be injected. So you could argue that it's like `$status` in the example above.

What's more, in order to make things simpler, we imply that a Number passed as a selector will be run by `PLURAL` by default, allowing us to do this:

```
unreadEmails = You have { $num -> 
  [0] no unread emails
  [one] one unread email
 *[other] many unread emails
}
```

How would you describe `$num` in that case vs the previous more explicit example?

Would you say that the explicit or implicit is easier for localizers to understand?